transmart_api.py

```diff
@@ -59,7 +62,16 @@ def get_hd_node_data(self, study, node_name, projection='all_data', genes = None
                 '&' + urllib.urlencode({'dataConstraints': {'genes': [{'names': genes}]}})
         hd_data = self._get_protobuf(hd_node_data_url, self._get_access_token())
         return hd_data
-    
+
+    def _get_json_post(self, url, access_token = None, hal = False):
+        headers = {}
+        headers['Accept'] = 'application/%s;charset=UTF-8' % ('hal+json' if hal else 'json')
+        if access_token is not None:
+            headers['Authorization'] = 'Bearer ' + access_token
+	req2 = urllib2.Request(url, "", headers)
```

Comment:
@laumadmar 
Thanks for the fix!
So basicaly there is small difference between `POST` code and `GET` code.
Namely in the way we initialize `Request` object.
`POST` code has `data=""`:
`urllib2.Request(url, "", headers)`
when `GET` code has `data=None`
`urllib2.Request(url, headers = headers)`
We might want to make `_get_json` receive one extra argument `http_method`. In such way we will reduce code repetition.